iphone/Classes/TiUIScrollableView.m

```diff
@@ -650,6 +650,31 @@ - (void)setPagingControlAlpha_:(id)args
   }
 }
 
+#if IS_SDK_IOS_14
+- (void)setPreferredIndicatorImage_:(id)args
```

Comment:
Do you need to do anything to expose the getter for the property?

Reply:
Getter will get its value from [proxy valueForKey:@"property_name"];
Ideally  we calculate get it's value from native API and return it. It should be like -
`- (TiBlob)preferredIndicatorImage {
     return ([[[TiBlob alloc] initWithImage:[[self pagecontrol] preferredIndicatorImage]] autorelease]);
}`

But problem is, we can pass Ti.Blob or String in setter, but in getter we can only get Ti.Blob. So it will be confusing. That's why no getter implementation